modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -3,15 +3,15 @@
 // * security/audit-log-view.adoc
 
 [id="nodes-nodes-audit-log-basic-viewing_{context}"]
-= Viewing the audit log
+= Viewing the audit logs
 
-You can view logs for the {product-title} API server or the Kubernetes API server for each master node.
+You can view logs for the OpenShift API server, Kubernetes API server, and OpenShift OAuth API server for each master node.
 
 .Procedure
 
-To view the audit log:
+To view the audit logs:
 
-. View the {product-title} API server logs
+* View the OpenShift API server logs
 
 .. If necessary, get the node name of the log you want to view:
```

Comment:
In this step, is the intention to get the node name and the name of the corresponding log? If so, you could change this line to something like the following, for clarity:

"Get the node name and the name of the log that you want to view."